packages/flutter/lib/src/material/material.dart

```diff
@@ -217,9 +221,29 @@ class Material extends StatefulWidget {
   /// Must be opaque. To create a transparent piece of material, use
   /// [MaterialType.transparency].
   ///
+  /// If inside a dark theme and [darkThemeOverlay] is [true], a semi-transparent
+  /// white will be composited on top this color to indicated the elevation.
+  ///
   /// By default, the color is derived from the [type] of material.
   final Color color;
 
+  /// Apply a semi-transparent white overlay on top of [color] for dark themes.
+  ///
+  /// If [true] a semi-transparent white overlay will be applied to the [color]
+  /// property when the surrounding theme's brightness is dark. The transparency
+  /// is based off of [elevation] as per the Material Dark theme specification.
+  ///
+  /// If [false] the [color] will be used unmodified.
+  ///
+  /// Defaults to [true].
+  ///
+  /// See also:
+  ///   * [Material.color]
+  ///   * [Material.elevation]
+  ///   * [ThemeData.brightness]
+  ///   * <https://material.io/design/color/dark-theme.html>
+  final bool darkThemeOverlay;
```

Comment:
I think a verb in the bool name is preferred, something like `applyDarkThemeOverlay`

Reply:
Good point. I was also thinking maybe it should include elevation as well. But then `applyDarkThemeElevationOverlay` is really getting to be a mouthful :).